PhotoFeed/app/src/main/java/com/mobileproto/jovanduy/photofeed/DbAccessor.java

```diff
@@ -0,0 +1,98 @@
+package com.mobileproto.jovanduy.photofeed;
+
+import android.content.ContentValues;
+import android.content.Context;
+import android.database.Cursor;
+import android.database.sqlite.SQLiteDatabase;
+import android.util.Log;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+
+/**
+ * Class to modify the database
+ */
+public class DbAccessor {
+
+    public FeedReaderDbHelper dbHelper;
+
+    public DbAccessor(Context context) {
+        this.dbHelper = new FeedReaderDbHelper(context);
+    }
+
+    /**
+     * add a url to the database
+     * @param url to be added to database
+     */
+    public void writeToDb(String url) {
+        SQLiteDatabase db = dbHelper.getWritableDatabase();
```

Comment:
You could speed up this function if you move this db declaration to where FeedReaderDbHelper is declared and initialize it in the constructor. That way you only would need to get the writable database once as opposed to every time you call this function.